DL_MODEL/person1.py

```diff
@@ -0,0 +1,94 @@
+import os
+import cv2
+import numpy as np
+import tensorflow as tf
+from matplotlib import pyplot as plt
+from tensorflow.keras import layers, models
+
+# Define paths to the dataset
+dataset_dir = "C:/Users/hp/Downloads/project/human detection dataset"
```

Comment:
Instead of Hardcoding this Line, Figure out a way that works in all System. Cuz when i stored the Code in Other Drive. There was Issue and I Had to Replace this Line of Code